ui/imports/shared/popups/send/views/TransactionModalFooter.qml

```diff
@@ -0,0 +1,67 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+import QtQml.Models 2.15
+
+import StatusQ.Controls 0.1
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Popups.Dialog 0.1
+
+import utils 1.0
+
+StatusDialogFooter {
+    id: root
+
+    property string maxFiatFees: "..."
+    property string totalTimeEstimate
+    property bool pending: true
+    property string nextButtonText: qsTr("Next")
```

Comment:
??
```suggestion
    property alias nextButtonText: nextButton.text
```

Reply:
This way is preferred for several reasons:
- immediately clear what the type is without following the alias reference to the target
- allows you to set a default value
- it can actually be used inside inline components (with `alias` it's crashing as I found out recently)

Reply:
As you prefer. However you can also a default value in the button text property!